notebooks/rag_eval_ragas.ipynb

```diff
@@ -9,7 +9,7 @@
         "# RAG pipeline evaluation using Ragas\n",
```

Comment:
<p>It seems like we dont cover context utilization in the notebook. Maybe replace it with "answer relevancy"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
We do :-)
![image](https://github.com/user-attachments/assets/df8aff3e-02bf-402d-a152-34ca4ab9cde1)